### PR TITLE
update to example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
Not perfect...
I have to check what are the CI result for the master branch
But manually restore to a fresh yunohost works

capture

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -113,21 +123,28 @@ $final_path/.pythonz/bin/pythonz install 3.6.6
...
 
-( cd $final_path
-virtualenv -p $($final_path/.pythonz/bin/pythonz locate 3.6.6) ve3
-ve3/bin/pip3 install -r requirements.txt )
+pushd $final_path
+    virtualenv -p $($final_path/.pythonz/bin/pythonz locate 3.6.6) ve3
+    ve3/bin/pip3 install -r requirements.txt
+	#Fix current websocket version error (2019-02-14)
+	ve3/bin/pip3 uninstall -y websockets
+	ve3/bin/pip3  install 'websockets>=6.0,<7.0'
```
Comment (maniackcrudelis) [inline scripts/install:131]: Sure that error happened every time ?
Never had that.

File: scripts/install [hunk 2]
```diff
@@ -113,21 +123,28 @@ $final_path/.pythonz/bin/pythonz install 3.6.6
 # INSTALL YUNORUNNER DEPENDENCIES
 #=================================================
 
-( cd $final_path
-virtualenv -p $($final_path/.pythonz/bin/pythonz locate 3.6.6) ve3
-ve3/bin/pip3 install -r requirements.txt )
+pushd $final_path
```
Comment (maniackcrudelis) [inline scripts/install:126]: Why not keeping the sub shell ?

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install:131]: I've tested 3 times on VMs installed from scratch... Each time the same issue... So I can imagine that will happen each time

File: scripts/install (hunk 2 above)
Comment (yalh76) [reply scripts/install:126]: I don't know, just more clear for me to read code

File: scripts/install (hunk 1 above)
Comment (maniackcrudelis) [reply scripts/install:131]: ok, guess that's ok then

File: scripts/install (hunk 2 above)
Comment (maniackcrudelis) [reply scripts/install:126]: Ok, no problem for me

**Comment by maniackcrudelis** [comment]
Let me know when it's ok for you

**Comment by yalh76** [comment]
yes, OK for me.

**Pull request merged**